.github/workflows/deploy_watermarker-source-code-docs.yml

```diff
@@ -5,12 +5,10 @@
 # that can be found in the LICENSE file.
 #
 
-name: "Watermarker: Generate API docs"
+name: "Deploy: Watermarker API docs"
 
 on:
-  push:
-    branches:
-      - main
+  workflow_dispatch:
```

Comment:
What is the reason to change this?

Reply:
Currently, this workflow conflicts with the Docusaurus deployment since both try to publish a website on GitHub pages. Changing this to `workflow_dispatch` turned off the workflow but made it still executable manually.

In the long run, the output from Dokka should be directly integrated into Docusaurus. To keep track of this, I created issue #68.